internal/terraform/rootmodule/path.go

```diff
@@ -10,3 +10,8 @@ func pathEquals(path1, path2 string) bool {
 	volume2 := filepath.VolumeName(path2)
 	return strings.EqualFold(volume1, volume2) && path1[len(volume1):] == path2[len(volume2):]
 }
+
+func ToLowerVolumePath(path string) string {
```

Comment:
I think this function doesn't need to be under `rootmodule`, but that's rather a nitpick - I can move it in a separate commit. 😄

Reply:
I don't know where to place it properly, please kindly take it over

Reply:
I think the best place for the function is the package where it is called from - I will move it there, no worries. 👍